### PR TITLE
dev/allow h26x wo container

### PR DESCRIPTION
**Pull request by bjornvolcker**
- Get first timestamp from SEI
- Validator now takes H.26x files without container

### CODE REVIEW

File: apps/signer/main.c [hunk 1]
```diff
@@ -204,9 +204,12 @@ main(gint argc, gchar *argv[])
   filesink = gst_element_factory_make("filesink", NULL);
 
   if (!filesrc || !demuxer || !parser || !muxer || !filesink) {
-    g_message(
-        "One or more standard elements in pipeline could not be found - check "
-        "your install.");
+    if (!filesrc) g_message("gStreamer element 'filesrc' not found");
+    if (!demuxer) g_message("gStreamer element '%s' not found", demux_str);
+    if (!parser) g_message("gStreamer element '%sparse' not found", codec_str);
```
Comment (lusikamalo) [inline apps/signer/main.c:209]: -> '%s' parser not found

Comment (bjornvolcker) [reply apps/signer/main.c:209]: But the gStreamer element is called `h264parse` and `h265parse`

**Review by lusikamalo — APPROVED** [review]
OK.

**Review by lusikamalo — APPROVED** [review]
OK.

**Pull request merged**